src/server/game/Server/WorldSocket.cpp

```diff
@@ -713,6 +713,13 @@ void WorldSocket::HandleAuthSessionCallback(std::shared_ptr<WorldPackets::Auth::
         digestKeyHash.UpdateData(buildInfo->Win64AuthSeed.data(), buildInfo->Win64AuthSeed.size());
     else if (account.Game.OS == "Mc64")
         digestKeyHash.UpdateData(buildInfo->Mac64AuthSeed.data(), buildInfo->Mac64AuthSeed.size());
+    else if (account.Game.OS == "MacA")
+        digestKeyHash.UpdateData(buildInfo->Mac64AuthSeed.data(), buildInfo->MacArmAuthSeed.size());
```

Comment:
```suggestion
        digestKeyHash.UpdateData(buildInfo->MacArmAuthSeed.data(), buildInfo->MacArmAuthSeed.size());
```

Reply:
Heh, woops. Didn't notice this as I'm still extracting the seed info.